core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -901,7 +922,6 @@ secret_configs:
       item: insights-ci-account
     ops-mirror.pem:
       field: cert-key.pem
-      item: mirror.openshift.com
```

Comment:
did you mean to remove this?

Reply:
No, It was by accident. I will change it. @smg247